R/timeSinceFire.R

```diff
@@ -35,11 +38,13 @@ timeSinceFireUI <- function(id, rastersNumber) {
 #' @param output Shiny server output object
 #' @param session Shiny server session object
 #' @param rasters Set of rasters to be displayed
+#' @param leafletZoomInit Initial leaflet zoom
+#' @param studyArea Size of study area. Options: "FULL", "EXTRALARGE", "LARGE", "MEDIUM", "NWT", "SMALL"
 #'
 #' @author Mateusz Wyszynski
 #'
 #' @export
-timeSinceFire <- function(input, output, session, rasters) {
+timeSinceFire <- function(input, output, session, rasters, leafletZoomInit, studyArea) {
```

Comment:
Maybe it's a good idea to add some default values for these two new parameters?

Reply:
Sure, done